packages/core/number/package.json

```diff
@@ -3,9 +3,9 @@
   "version": "0.0.6",
   "license": "MIT",
   "source": "src/index.ts",
-  "main": "dist/index.js",
-  "module": "dist/index.module.js",
-  "types": "dist/index.d.ts",
+  "main": "dist/radix-ui-number.cjs.js",
+  "module": "dist/radix-ui-number.esm.js",
+  "types": "dist/radix-ui-number.cjs.d.ts",
```

Comment:
Technically the `"types"` field is not required so it could be removed but there are still some reasons to keep it around. See https://github.com/preconstruct/preconstruct/issues/259